### ISSUE TITLE
Circularity in the test. 

### ISSUE THREAD

**Issue by Yehouda** [Shinmera/pathname-utils#12]
Trying to run the parachute tests goesinot infinite loop (causing a stack overflow error), because there is circularity in the test definitions, because the test to cativate circularly in  this path:

       predicates -> subpath-p -> enough-pathname -> predicates. 

**subpath-p** is a child of **predicates**.

     predicates -> subpath-p

**subpath-p** depends on **enough-pathname**

     subpath-p -> enough-pathname

**enough-pathname** child of **queries**, queries depends on **predicates**
      => **enough-pathname** depends on **predicates**. 

    enough-pathname -> predicates 

I "fixed" it by by changing **enough-pathname** to be a child of **pathname-utils**, rather than **queries**, so at least
it doens't get the error (gets various failures).

**Comment by Shinmera** [comment]
The tests go through just fine for me without any changes.

**Comment by Yehouda** [comment]
So somehow it doesn't loop for you. How  is Parachute supposed to not loop in this case? 

Do you get any failures? Maybe if it doesn't get failures it gets out of the loop. 

May also be version of parachute, I got  from quicklisp:

parachute-20230214-git
pathname-utils-20221106-git

Also may be the way I run it. LispWorks on freebsd or linux:
```
CL-USER 24 > (setq res (PARACHUTE:TEST  "PATHNAME-UTILS-TEST" ; package name
                                       :report 'parachute::quiet))
(setq res (PARACHUTE:TEST  "PATHNAME-UTILS-TEST" ; package name
                                       :report 'parachute::quiet))

Stack overflow (stack size 15997).
  1 (continue) Extend stack by 50%.
  2 (abort) Return to debug level 0.
  3 Return to top-level loop.
  4 Return from multiprocessing.
```

**Comment by Shinmera** [comment]
Yes, you're missing this patch: https://github.com/Shinmera/parachute/commit/bd072b0e4d0ff3ee2201eca3eb28c287686ab49e

**Comment by Yehouda** [comment]
That fixes it.
